src/Lykke.Service.OperationsHistory.Job/Modules/JobModule.cs

```diff
@@ -81,5 +116,39 @@ private void RegisterAzureRepositories(ContainerBuilder builder)
                         _dbSettingsManager.ConnectionString(x => x.DataConnString), "OperationsHistory", _log)))
                 .As<IHistoryLogEntryRepository>();
         }
+        
+        private void RegisterDictionaryEntities(ContainerBuilder builder)
+        {
+            builder.Register(c =>
+            {
+                var ctx = c.Resolve<IComponentContext>();
+                return new CachedDataDictionary<string, Asset>(
```

Comment:
This code will lead to errors when new assets / asset-pairs will arrive.
Please use ICacheManager / OnDemandDataCache from Lykke.Common for caching purpose. Or use cached implementation of AssetService.